File: src/main/java/com/etour/app/invoice/dto/InvoiceAmountDTO.java
package com.etour.app.invoice.dto;

import java.math.BigDecimal;

public class InvoiceAmountDTO {

    private BigDecimal tourAmount;
    private BigDecimal taxAmount;
    private BigDecimal totalAmount;
    private String currency;

    public InvoiceAmountDTO() {}

    public BigDecimal getTourAmount() {
        return tourAmount;
    }

    public void setTourAmount(BigDecimal tourAmount) {
        this.tourAmount = tourAmount;
    }

    public BigDecimal getTaxAmount() {
        return taxAmount;
    }

    public void setTaxAmount(BigDecimal taxAmount) {
        this.taxAmount = taxAmount;
    }

    public BigDecimal getTotalAmount() {
        return totalAmount;
    }

    public void setTotalAmount(BigDecimal totalAmount) {
        this.totalAmount = totalAmount;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }
}

File: dotnet/backend/DTO/Invoice/InvoiceAmountDTO.cs
namespace Etour_Backend_dotnet.DTO.Invoice;

public class InvoiceAmountDTO
{
    public decimal TourAmount { get; set; }
    public decimal TaxAmount { get; set; }
    public decimal TotalAmount { get; set; }
    public decimal PaidAmount { get; set; }
    public decimal BalanceAmount { get; set; }
}
